docs/index.html

```diff
@@ -10,6 +10,7 @@
     <meta name="description" content="">
     <meta name="viewport" content="width=device-width">
     <link rel="stylesheet" href="blueprint-landing.css">
+    <link rel="icon" href="resources/img/favicon.png" sizes="32x32" type="image/png">
```

Comment:
`docs/` directory should not be edited by hand, only by the build.
this must be identical to `packages/landing/src/index.html` below (isn't currently).